### DISCUSSION TITLE
Run seeding for dev and prod

### DISCUSSION THREAD

**Discussion by bmadone** [remix-run/remix#3175] [Q&A]
I use indie stack, and can't  find any information how to seed db that already hosted in fly.io, any references?

**Comment by asciant** [comment]
I use these within my `start.sh` to achieve that. I usually just uncomment whichever I don't need, and if the schema is stable, I will uncomment them all.

```
# This file is how Fly starts the server (configured in fly.toml). Before starting
# the server though, we need to run any prisma migrations that haven't yet been
# run, which is why this file exists in the first place.
# Learn more: https://community.fly.io/t/sqlite-not-getting-setup-properly/4386

#!/bin/sh

set -ex
# npx prisma db push --accept-data-loss
# npx prisma db push --force-reset && npx prisma db seed
# npx prisma db seed
npm run start
```

**Reply by amypellegrini** [reply]
Do you have any memory issues in fly? I'm trying to do this but it crashes my instance due to lack of memory. It's also missing dependencies like typescript and ts-node which I had to move from devDependencies.

**Reply by asciant** [reply]
Yes, I had to increase the memory in the scale tab to 512mb (which is still free).

To avoid including TS in the project dependencies, I rewrote the seed file in JS `seed.mjs` and updated the seed script in package.json to:

```
  "prisma": {
    "seed": "node --experimental-loader=module prisma/seed.mjs"
  }
```
